File: sms-sender-archetype/src/main/resources/archetype-resources/src/main/java/__pluginName__/__pluginName__SmsSender.java
package ${package}.${pluginName};

import se.curity.identityserver.sdk.service.sms.SmsSender;

public final class ${pluginName}SmsSender implements SmsSender
{
    private final ${pluginName}SmsConfig _configuration;

    public ${pluginName}SmsSender(${pluginName}SmsConfig configuration)
    {
        _configuration = configuration;
    }

    @Override
    public boolean sendSms(String toNumber, String msg)
    {
        // TODO: Implement sending an sms.
        // Check out the examples on GitHub: https://github.com/search?q=sms+org%3Acurityio&type=Repositories

        return true;
    }
}

File: kotlin-sms-sender-archetype/src/main/resources/archetype-resources/src/main/kotlin/__pluginName__/__pluginName__SmsSender.kt
package ${package}.${pluginName}

import se.curity.identityserver.sdk.service.sms.SmsSender

class ${pluginName}SmsSender(private val _configuration: ${pluginName}SmsConfig): SmsSender
{
    override fun sendSms(toNumber: String, msg: String): Boolean
    {
        // TODO: Implement sending an sms.
        // Check out the examples on GitHub: https://github.com/search?q=sms+org%3Acurityio&type=Repositories

        return true
    }
}
